modules/workers/nodepools.tf

```diff
@@ -80,7 +80,7 @@ resource "oci_containerengine_node_pool" "workers" {
   )
 
   node_eviction_node_pool_settings {
-    eviction_grace_duration              = format("PT%sM", each.value.eviction_grace_duration)
+    eviction_grace_duration              = format("PT%sS", each.value.eviction_grace_duration)
```

Comment:
why was this changed away from `PT%sM` to `PT%sS` ? This is causing a permadiff in RC4.